plugin/trino-redshift/src/test/java/io/trino/plugin/redshift/TestRedshiftCastPushdown.java

```diff
@@ -227,6 +228,25 @@ public void testJoinPushdownWithNestedCast()
                 .isFullyPushedDown();
     }
 
+    @Test
+    public void testCaseSensitiveComparison()
+    {
+        String schemaAndTable1 = TEST_SCHEMA + "." + "case_sensitive_1_" + randomNameSuffix();
+        String schemaAndTable2 = TEST_SCHEMA + "." + "case_sensitive_2_" + randomNameSuffix();
+
+        executeInRedshift("CREATE TABLE " + schemaAndTable1 + "(c_casesensitive VARCHAR(10) COLLATE CASE_SENSITIVE, c_caseinsensitive VARCHAR(10) COLLATE CASE_INSENSITIVE)");
+        executeInRedshift("CREATE TABLE " + schemaAndTable2 + "(c_casesensitive VARCHAR(20) COLLATE CASE_SENSITIVE, c_caseinsensitive VARCHAR(20) COLLATE CASE_INSENSITIVE)");
```

Comment:
Can we use `TestTable` here as well?

Reply:
It uses Trino table definition but this table to be created with Redshift native syntax to support COLLATE while creating a table.

Reply:
Where's the Trino table definition in [TestTable](https://github.com/trinodb/trino/blob/master/testing/trino-testing/src/main/java/io/trino/testing/sql/TestTable.java)? It should allow any column definitions:
```java
        new TestTable(
                TestingRedshiftServer::executeInRedshift,
                schemaAndTable1,
                "(c_casesensitive VARCHAR(10) COLLATE CASE_SENSITIVE, c_caseinsensitive VARCHAR(10) COLLATE CASE_INSENSITIVE)")
```

Reply:
thanks for correcting me. I had tried with trino executor instead of redshift executor.